clients/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metaplex-foundation/mpl-toolbox",
-  "version": "0.9.2",
+  "version": "0.10.0",
```

Comment:
I think a patch bump is okay now since we're no longer adding a breaking change.